src/commands/sbom/validation.ts

```diff
@@ -48,10 +55,23 @@ export const validateSbomFileAgainstSchema = (path: string, ajv: Ajv, debug: boo
       process.stdout.write('File is a valid CycloneDX 1.4 file\n')
     }
 
-    if (isValid14 || isValid15) {
+    if (isValid14 || isValid15 || isValid16) {
       return true
     }
 
+    // show the errors
+    if (!isValid16) {
+      const errors15 = validateFunctionCycloneDx16.errors || []
+
+      if (debug) {
+        errors15.forEach((message) => {
+          process.stderr.write(
+            `Error while validating file against CycloneDX 1.6: ${path}, ${message.schemaPath}: ${message.instancePath} ${message.message}\n`
```

Comment:
Looking at this message, it seems the same across all versions except for the version itself, shouldn't we have the logic only once which takes the version in parameter instead of duplicating the logic for each version ?